sparkle/src/components/Modal.tsx

```diff
@@ -29,7 +29,7 @@ const modalStyles = {
   [ModalType.Side]: {
     containerClasses: "s-h-full s-ml-auto",
     panelClasses:
-      "s-h-full s-max-h-full s-shadow-xl s-border s-border-structure-100",
+      "s-h-full s-max-h-full s-shadow-xl s-border s-border-structure-100 s-w-full s-max-w-full s-r-0",
```

Comment:
What's `r-0`?

Reply:
technically it should be 'right: 0px;'
indeed doesn't seem to have any effect, removing & merging